tests/test_material.py

```diff
@@ -73,8 +73,9 @@ def test_error_table_negative_val(testdir, caplog):
     f = os.path.join(testdir, 'test_inputs', 'custom_mat-3.csv')
     # with pytest.raises(SystemExit):
     #     Material('badbad', from_file=f)
-    Material('badbad', from_file=f)
-    assert 'Negative values detected in material data ' in caplog.text
+    with pytest.raises(SystemExit):
```

Comment:
According to the modification of the scope of this test, I'd suggest to change the name of the method.